hermes-api/src/main/java/pl/allegro/tech/hermes/api/OfflineRetransmissionTask.java

```diff
@@ -43,6 +45,10 @@ public String getSourceTopic() {
         return request.getSourceTopic();
     }
 
+    public String getSourceViewPath() {
```

Comment:
Maybe we can use Optional to indicate that one of the values `sourceView`/`sourceTopic` can be emtpy

Reply:
Goodpoint on it